### PR TITLE
Add the API call to search Users with a given tag

### PR DESCRIPTION
**Pull request by StephaneJuban**
Hi @ches 

This is my (first) contribution to this repo.
It add the GET /tags/:id/users feature.

Regards,
Stéphane

### CODE REVIEW

File: spec/integration/users_spec.rb [hunk 1]
```diff
@@ -28,5 +29,19 @@
...
+    investors = client.user_tags(tag_id, :investor => 'by_residence')
+    
+    # Get the first investor user profile
+    user = client.get_user(investors.users[0].id)
+    
+    # Check his locations tags
+    location_match = false
+    user.locations.each do |location|
+      location_match = true if location.id == tag_id
+    end
```
Comment (ches) [inline spec/integration/users_spec.rb]: Not a big deal, but you could use `user.locations.any? { |loc| loc.id == tag_id }` here :smile:

**Comment by ches** [comment]
Thanks Stéphane! Sorry the tests were such a pain to get going.

File: spec/integration/users_spec.rb (hunk 1 above)
Comment (StephaneJuban) [inline spec/integration/users_spec.rb:43]: Thanks for the tip! Still not knowing all the tips & tricks of Ruby (coming from C dev :D)
I will update it if I do more contributions.

**Comment by StephaneJuban** [comment]
It's ok, it makes me dig deeper and understand the test process ;)

**Pull request merged**